inspec.yml

```diff
@@ -6,4 +6,4 @@ copyright: spaterson@chef.io,russell.seymour@turtlesystems.co.uk
 copyright_email: spaterson@chef.io,russell.seymour@turtlesystems.co.uk
 version: 1.81.19
 license: Apache-2.0
-inspec_version: '>= 4.18.108'
+inspec_version: '>= 4.56.17'
```

Comment:
Consider adding a comment explaining why this version is needed

Reply:
done